projects/dsp-ui/src/lib/viewer/views/property-view/property-view.component.ts

```diff
@@ -10,33 +18,70 @@ import { ReadResource, SystemPropertyDefinition } from '@dasch-swiss/dsp-js';
 })
 export class PropertyViewComponent implements OnInit {
 
-  @ViewChild('displayEdit') displayEditComponent: DisplayEditComponent;
+    @ViewChild('displayEdit', { static: false}) displayEditComponent: DisplayEditComponent;
+    @ViewChild('addValue', { static: false}) addValueComponent: AddValueComponent;
+    /**
+     * Parent resource
+     *
+     * @param (resource)
+     */
+    @Input() parentResource: ReadResource;
 
-  /**
-   * Parent resource
-   *
-   * @param (resource)
-   */
-  @Input() parentResource: ReadResource;
+    /**
+     * Array of property object with ontology class prop def, list of properties and corresponding values
+     *
+     * @param (propArray)
+     */
+    @Input() propArray: PropertyInfoValues[];
 
-  /**
-   * Array of property object with ontology class prop def, list of properties and corresponding values
-   *
-   * @param (propArray)
-   */
-  @Input() propArray: PropertyInfoValues[];
+    /**
+     * Array of system property object with list of system properties
+     *
+     * @param (propArray)
+     */
+    @Input() systemPropArray: SystemPropertyDefinition[];
 
-  /**
-   * Array of system property object with list of system properties
-   *
-   * @param (propArray)
-   */
-  @Input() systemPropArray: SystemPropertyDefinition[];
+    addButtonIsVisible: boolean; // used to toggle add value button
+    addValueFormIsVisible: boolean; // used to toggle add value form field
+    propID: string; // used in template to show only the add value form of the corresponding value
+    readOnlyProp: boolean; // used in template to not show an "add" button for properties we do not yet have a way to create/edit
 
-  constructor() { }
+    eventBusSubscription: Subscription;
 
-  ngOnInit() {
+    constructor(private valueTypeService: ValueTypeService,
```

Comment:
valueTypeService is not used, it can be removed

Reply:
removed in c212919